genie-agent/src/main/java/com/netflix/genie/agent/execution/services/impl/JobSetupServiceImpl.java

```diff
@@ -695,7 +695,7 @@ String composeScript() {
             sb
                 .append("# Launch the command")
                 .append(NEWLINE)
-                .append(this.commandLine).append(" &").append(NEWLINE)
+                .append(this.commandLine).append(" <&0 &").append(NEWLINE)
```

Comment:
Any doc on this `<&0`? I googled but only found vague explanations.

Reply:
It's just bash redirection (https://www.tldp.org/LDP/abs/html/io-redirection.html, https://www.gnu.org/software/bash/manual/html_node/Redirections.html).

Specifically, the `stdin` of `bash run` process is forwarded to the actual command.
So in case of interactive CLI commands, when you type `select * from foo`, that line goes to command, rather than bash.

Reply:
Understand the redirection part, but never seen `<&0` to redirect parent stdin to child. Good to know.